src/pages/plugins/concepts/manifest/index.md

```diff
@@ -0,0 +1,788 @@
+            <p><inlineCode>{'{'}{'}'}</inlineCode> (no strings)</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>version</inlineCode> *</td>
+        <td><inlineCode>string</inlineCode></td>
+        <td>required</td>
+        <td>
+            <p>The version indicates the plugin"s version. The string has a format of major.minor.patch.</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>main</inlineCode></td>
+        <td><inlineCode>string</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>Indicates the primary JavaScript or HTML file, relative to the plugin"s installation directory. Supports HTML and JS files, such as <inlineCode>index.html</inlineCode> and <inlineCode>index.js</inlineCode>. If not specified (for deprecations), <inlineCode>main.js</inlineCode> is used.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>"main.js"</inlineCode></p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>icons</inlineCode></td>
+        <td><inlineCode>IconDefinition[]</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>An array of icons representing the overall plugin or panel icon. The host application uses these icons to present the plugin to the user. If the icons array is missing, a default icon will be used.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>[]</inlineCode> (no icons)</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>host</inlineCode> *</td>
+        <td><inlineCode>HostDefinition</inlineCode></td>
+        <td>required</td>
+        <td>
+            <p>The host object indicates the plugin"s compatibility with the host. Incompatible plugins will:</p>
+            <ul>
+                <li>fail to install if attempted in the given host</li>
+                <li>be invisible in the in-app plugin marketplace for the given host</li>
+                <li>be unavailable for update if the update is no longer compatible.</li>
+            </ul>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>entrypoints</inlineCode> *</td>
+        <td><inlineCode>EntrypointDefinition[]</inlineCode></td>
+        <td>required</td>
+        <td>
+            <p>The entrypoints array defines the entrypoints that the plugin provides.</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>requiredPermissions</inlineCode></td>
+        <td><inlineCode>PermissionsDefinition</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>Indicates the various permissions this plugin requires. These permissions are required before accessing certain API surfaces. Without these permissions in the manifest, API access may fail or throw an error. Some permissions require user consent.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>{'{'}{'}'}</inlineCode> (no permissions)</p>
+        </td>
+    </tr>
+    <tr>
```

Comment:
This is not available in Id yet.

Reply:
Will it be available once it goes to GA? I wouldn't want to remove it (since the document is meant to be used across host apps). If it'll be available in GA, I would suggest that we keep it in there and make sure to mention in the prerelease that this is (_current limitations_) not available but will be for GA. If it isn't available once plugins for ID go live, we ... have a bigger problem, I think (in that case, I would add a note about compatibility here, but generally speaking, I feel like that would be pretty bad).

Reply:
No, it wont be available in InDesign until later this year..

Reply:
Wait ... which one are we talking about? `permissions` or `addon`?

Reply:
Just addon

Reply:
Ok. Still unfortunate, but not as bad as the alternative, then. I'll add a note.